python/pyspark/rdd.py

```diff
@@ -1440,9 +1666,9 @@ def mean(self):
         >>> sc.parallelize([1, 2, 3]).mean()
         2.0
         """
-        return self.stats().mean()
+        return self.stats().mean()  # type: ignore[return-value]
```

Comment:
We might revisit `StatCounter` later (maybe it should be generic, but it is tricky to do right if input is empty), but for now let's use ignores.